packages/base/src/theming/CustomStyle.js

```diff
@@ -16,14 +17,31 @@ const fireCustomCSSChange = tag => {
 	return eventProvider.fireEvent(CUSTOM_CSS_CHANGE, tag);
 };
 
-const customCSSFor = {};
+const customCSSFor = getSharedResource("CustomStyle.customCSSFor", {});
+
+// Listen to the eventProvider, in case other cores fire this event and this core
+// would therefore need to reRender the ui5 webcomponents, but don't rerender if it was
+// this core that fired the event to begin with.
```

Comment:
**[suggestion (non-blocking)](https://conventionalcomments.org/#labels):**

```suggestion
// Listen to the eventProvider, in case other copies of this CustomStyle module fire this
// event, and this copy would therefore need to reRender the ui5 webcomponents; but
// don't reRender if it was this copy that fired the event to begin with.
```